content/docs/user-guide/building/requirements.md

```diff
@@ -0,0 +1,40 @@
+# Requirements
+
+Get `pip` based and `unix` based requirements for your model.
+
+## Description
+
+`mlem` can export the requirements needed for a model using the
+`mlem build model requirements` command, where `model` is the path to model
+saved via `mlem`
+
+```cli
+$ mlem build model requirements
+⏳️ Loading model from model.mlem
+scikit-learn==1.0.2
+pandas==1.4.2
+numpy==1.22.3
+```
+
+One can also save these `requirements` to generate a `requirements.txt` file
+using the `-c target` option which allows us to pass a path of the file i.e.
+
+```cli
+$ mlem build model requirements -c target="requirements.txt"
```

Comment:
```suggestion
$ mlem build model requirements -c target requirements.txt
```

I think we show it like this everywhere in docs

Reply:
This is because it pertains to the current main branch. For the new 0.3 release, it should be `--target requirements.txt` without the `-c`. I guess you meant this because your suggestion mixes both old and new approach. I will change according to the new.

Reply:
done